### DISCUSSION TITLE
MSVC toolset v141 build errors

### DISCUSSION THREAD

**Discussion by AnloWho** [DiligentGraphics/DiligentCore#626] [Q&A] [answered]
From the readme it is mentioned to use the following for building:
```cmake -S . -B ./build/Win64 -G "Visual Studio 17 2022" -A x64```
However, in cmake I want to specify -T v141 to make use of an older toolset. However, when compiling this in VS2022 I get build errors for both Debug and Release builds. Having the project configured for toolset v143 is working. I'm just wondering if DiligentCore is supposed to work with v141 or not?

**Comment by TheMostDiligent** [comment, marked as answer]
We support Visual Studio 2019 toolset, but older versions are not guaranteed to work.

**Reply by AnloWho** [reply]
Ok. good to know. then I'll stick with the v143 of VS2022.
